### PR TITLE
Disabled publish button

### PR DESCRIPTION
**Pull request by kanchan-bluebash**
Issue:- #216 
When user creates an account on publiCV and **doesn't confirm** their account, they wont be able to **publish** their CV and will be redirected to a link, from where they can confirm their account and can publish their CV so that it will be visible openly.

https://user-images.githubusercontent.com/81285549/133414915-a8633bf3-4537-4ec9-a675-72d8c530c435.mp4

### CODE REVIEW

File: config/locales/tooltips.en.yml [hunk 1]
```diff
@@ -8,3 +8,5 @@ en:
     extras:
       interests: 'Short description of both your job-related and job-unrelated interests.'
       future_plans: 'What would you want to accomplish in the next 3 years, from a professional perspective?'
+    buttons:
+      publish: "<span class='text-white'>You need to confirm your PubliCV account first. <br>Click <a class='text-primary bg-light font-weight-bold text-decoration-none' href='/users/confirmation/new'>here</a> to confirm</span>"
```
Comment (vishal-bluebash) [inline config/locales/tooltips.en.yml:12]: Update route to be dynamic

File: app/views/cvs/_cv_controls.html.erb [hunk 2]
```diff
@@ -2,12 +2,18 @@
...
 
         <% if @cv.published? %>
-          <%= link_to 'Unpublish', cv_path("cv[published]" => "0"), method: :put, class: 'btn btn-outline-primary mx-2' %>
+          <%= link_to 'Unpublish', cv_path("cv[published]" => "0"), method: :put, class: 'btn btn-outline-primary mx-3' %>
         <% else %>
-          <%= link_to 'Publish', cv_path("cv[published]" => "1"), method: :put, class: 'btn btn-outline-primary mx-2' %>
+          <% if current_user.confirmed? %>
+            <%= link_to 'Publish', cv_path("cv[published]" => "1"), method: :put, class: 'btn btn-outline-primary mx-3' %>
+          <% else %>
+            <button class="btn btn-outline-primary mx-3 disabled" data-toggle="tooltip" data-placement="top" title="<%= t('tooltips.buttons.publish') %>">
```
Comment (vishal-bluebash) [inline app/views/cvs/_cv_controls.html.erb:13]: We can use button tag

**Comment by vishal-bluebash** [comment]
PR looks good, some minor changes, Once done assign it to Duarme  for review so that we can have translations for tooltips 

@bb-varun @kanchan-bluebash

**Review by vishal-bluebash — APPROVED** [review]

File: config/locales/tooltips.en.yml [hunk 3]
```diff
@@ -8,3 +8,5 @@ en:
     extras:
       interests: 'Short description of both your job-related and job-unrelated interests.'
       future_plans: 'What would you want to accomplish in the next 3 years, from a professional perspective?'
+    buttons:
+      publish: "<span class='text-white'>You need to confirm your PubliCV account first. <br>Click <a class='text-primary bg-light font-weight-bold text-decoration-none' href='%{url}'>here</a> to confirm</span>"
```
Comment (duarme) [inline config/locales/tooltips.en.yml:12]: when i18n contains HTML, the key should end with `_html`, therefore this should be: `publish_html`

Comment (duarme) [reply config/locales/tooltips.en.yml:12]: Then an improved content could be:

```
publish_html: "<span class='text-white'>In order to publish your publiCV, you need to confirm you account first. <br>Click <a class='text-primary bg-light font-weight-bold text-decoration-none' href='%{url}'>here</a> to confirm</span>"

```

**Pull request merged**